src/phantom/action_result.py

```diff
@@ -0,0 +1,52 @@
+from phantom.app import APP_ERROR
```

Comment:
By adding a very small amount of simple logic to these stubs, we can make them behave like the real thing and eliminate a lot of the need for mocks. All these methods exist on these classes in the real world.